syncmaster/schemas/v1/queue.py

```diff
@@ -1,21 +1,31 @@
 # SPDX-FileCopyrightText: 2023-2024 MTS PJSC
 # SPDX-License-Identifier: Apache-2.0
-from pydantic import BaseModel, Field, constr
+from pydantic import BaseModel, Field, constr, model_validator
+from pydantic.json_schema import SkipJsonSchema
 
 from syncmaster.schemas.v1.page import PageSchema
 
 
 class CreateQueueSchema(BaseModel):
     name: constr(max_length=128, pattern=r"^[-_a-zA-Z0-9]+$") = Field(  # noqa: F722
         ...,
-        description="Queue name",
+        description="Queue name that allows letters, numbers, dashes, and underscores",
     )
     group_id: int = Field(..., description="Queue owner group id")
     description: str = Field(default="", description="Additional description")
+    slug: SkipJsonSchema[str] = Field(description="Generated slug for unique queue identification")
```

Comment:
Probably [computed field](https://docs.pydantic.dev/2.0/usage/computed_fields/) can be used instead